src/EA.Weee.Web/Areas/AATFReturn/Mappings/ToViewModel/AddressUtilities.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace EA.Weee.Web.Areas.AatfReturn.Mappings.ToViewModel
+{
+    using EA.Weee.Core.AatfReturn;
+
+    public class AddressUtilities
+    {
+        public string AddressConcatenate(AddressData addressData)
+        {
+            var address = string.Empty;
+
+            address = addressData.Address1;
+
+            address = addressData.Address2 == null ? address : StringConcatenate(address, addressData.Address2);
+
+            address = StringConcatenate(address, addressData.TownOrCity);
+
+            address = addressData.CountyOrRegion == null ? address : StringConcatenate(address, addressData.CountyOrRegion);
+
+            address = addressData.Postcode == null ? address : StringConcatenate(address, addressData.Postcode);
+
+            address = StringConcatenate(address, addressData.CountryName);
+
```

Comment:
Add unit tests around this, i.e. create a AddressUtilitiesTests class

Reply:
Tests created for this and tonnage utilities